collections/e12c_motoway.yaml

```diff
@@ -20,9 +20,46 @@ License: |
  [GNU General Public License v3.0](https://github.com/hfisser/Truck_Detection_Sentinel2_COVID19/blob/master/LICENSE)
 Resources:
  - Group: Sentinel Hub Resources
-   EndPoint: shservices.mundiwebservices.com
+ - EndPoint: shservices.mundiwebservices.com
+   Name: Sentinel Hub
+   Role: processor
    Type: byoc-6d697640-a962-4eb0-8a59-30a8cb61dc0f
    CollectionId: 6d697640-a962-4eb0-8a59-30a8cb61dc0f
-
+   Primary: True
```

Comment:
`True` should probably be lowercase. 

Also in other files, I won't write separate comments for all of them.

Reply:
Changed in all files